cmake/ConfigureCompiler.cmake

```diff
@@ -351,6 +355,14 @@ else()
       -fno-builtin-realloc
       -fno-builtin-free)
   endif()
+  # SQLite won't build with -ffast-math
+  if (ICX)
+    add_compile_options($<$<COMPILE_LANGUAGE:C>:-fno-fast-math>)
+    add_compile_options($<$<COMPILE_LANGUAGE:C>:-ffp-contract=on>)
+    add_compile_options($<$<COMPILE_LANGUAGE:CXX>:-ffp-contract=on>)
+    # No libc++ and libc++abi in Intel LIBRARY_PATH
+    add_link_options(-L/usr/local/lib)
```

Comment:
I think [find_library](http://cmake.org/cmake/help/v3.23/command/find_library.html) is what you want to use here. You can pass `/usr/local/lib` as a hint. It would also be better to then use `link_libraries` instead of passing a link option

Reply:
Thank you, @sfc-gh-mpilman! Done.